extra-files/cloud_setup/tool_data_table_conf.xml

```diff
@@ -0,0 +1,146 @@
+<?xml version="1.0"?>
+<!--
+    This file is maintained by Ansible - CHANGES WILL BE OVERWRITTEN
+-->
+<tables>
+    <!-- Locations of all fasta files under genome directory -->
+    <table name="all_fasta" comment_char="#">
+        <columns>value, dbkey, name, path</columns>
+        <file path="/cvmfs/data.galaxyproject.org/byhand/location/all_fasta.loc" />
```

Comment:
FYI I'm trying to set up jenkins (with https://github.com/jmchilton/galaxy-community-qa) to test deployments and build flavour containers automatically.

For the fuse/CVMFS issue we could probably resort to a compose setup, where a privileged fuse container mounts the filesystem and provides that as a volume. Or we just mount /cvmfs before starting the container and bindmount it into the container? Anyway, that's for the next PR.